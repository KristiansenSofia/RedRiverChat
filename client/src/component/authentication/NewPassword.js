import React, { Component } from 'react';

// Import styles. newPasswordStyles for all imported components with a style attribute and CSS-file for classNames and id.
import {newPasswordStyles} from "../../styles/AuthStyles";
import '../../styles/Styles.css'

/**
 *  newPassword-component.
 *
 *  @author Jimmy
 */

class NewPassword extends Component {
    render() {
        return (
            <div className="NewPassword">
                <p>Glömt lösenord</p>
            </div>

        );
    }
}

export default NewPassword;