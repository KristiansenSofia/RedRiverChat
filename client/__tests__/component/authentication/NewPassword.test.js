import React from 'react';
import NewPassword from '../../../src/component/authentication/NewPassword';
import renderer from 'react-test-renderer';

test('NewPassword renders without crashing', () => {
  const tree = renderer.create(<NewPassword />);
  expect(tree.toJSON()).toMatchSnapshot();
});
