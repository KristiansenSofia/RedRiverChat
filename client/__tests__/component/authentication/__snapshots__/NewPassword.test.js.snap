// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewPassword renders without crashing 1`] = `
<div
  className="NewPassword"
>
  <p>
    Glömt lösenord
  </p>
</div>
`;
